### DISCUSSION TITLE
Transparent or regular proxy?

### DISCUSSION THREAD

**Discussion by dspiegelman** [synchronizing/mitm#22] [Q&A] [answered]
This looks awesome! I would love to use it but I require a transparent proxy. Does mitm function as a transparent proxy or a regular proxy? Thank you.

**Comment by synchronizing** [comment, marked as answer]
Hey @dspiegelman. `mitm` is indeed a transparent proxy. No headers or identification is added to the clients and/or server requests/responses.
